action/depositreward_test.go

```diff
@@ -7,28 +7,9 @@ import (
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/iotexproject/iotex-address/address"
-	"github.com/iotexproject/iotex-proto/golang/iotextypes"
 	"github.com/stretchr/testify/require"
 )
 
-func TestDepositRewardProto(t *testing.T) {
-	r := require.New(t)
-
-	rp := &DepositToRewardingFund{}
-
-	p := rp.Proto()
-	r.NotNil(p)
-	r.IsType(&iotextypes.DepositToRewardingFund{}, p)
-	r.Nil(p.Data)
-
-	rp.amount = big.NewInt(100)
-	rp.data = []byte{1}
-	p = rp.Proto()
-	r.NotNil(p)
-	r.EqualValues([]byte{1}, p.Data)
-	r.EqualValues("100", p.Amount)
-}
```

Comment:
we can keep this test?

Reply:
`Proto()` related tests already in `rewardmsg_test.go`